### ISSUE TITLE
Documentation on workspace file XML tags (seasonal adjustment parameter settings)

### ISSUE THREAD

**Issue by a-s-russo** [jdemetra/jdplus-main#423]
Is there documentation on all possible XML tags (that is, all seasonal adjustment parameter settings as per the Specifications panel) that could possibly populate a workspace file using the X13 algorithm?

If not, could we please request such documentation, specifically doco that contains:
- all possible XML tags/parameters (along with very brief descriptions of each one)
- default values for all tags/parameters (along with very brief descriptions of the sets of allowable values)
- which classes/variables in the Java code correspond to which tags/parameters

It is worth mentioning that we have struggled to map settings from our existing seasonal adjustment package to JDemetra+ data items in the absence of such documentation, both methodologically (first and second points above) and technologically (third point above). 

This has been exacerbated by parameters in the workspaces not appearing in the files if they assume default values. That is, they are 'hidden' from us, making it hard to understand the set of all possible parameter settings that we could choose from; ideally, all tags would always be populated in each workspace file regardless of value. We have also had to use a lot of educated guessing and trial and error to understand the corresponding classes/variables in the Java code.

**Comment by Immurb** [comment]
Every X13 specification mapping should be in the [information part of the code](https://github.com/jdemetra/jdplus-main/tree/develop/jdplus-main-base/jdplus-x13-base-parent/jdplus-x13-base-information/src/main/java/jdplus/x13/base/information) and correspond with the classes in the api part.

I'm not sure if the xml is made to be written from outside JDemetra and therefore has no additional documentation, but @charphi should be more knowledgeable on that part.

**Comment by palatej** [comment]
Indeed. The current xml is for internal purposes. Moreover, it doesn't correspond to an actual xml schema; it just defines a tree structure with a small set of possible nodes. So, you are not able to validate your xml (except for the syntax).

I made for v2 a true xml schema, but it was really cumbersome and I think that nearly nobody used it. See [jtss/src/main/resources](https://github.com/jdemetra/jdemetra-core/tree/develop/jtss/src/main/resources) for more details.

In the future, if we want to build WEB/GRPC-services, we probably should use well-defined and documented protocols (based for instance on protos files). Developments for the medium term.
